fastlane/lib/fastlane/actions/sonar.rb

```diff
@@ -98,9 +98,9 @@ def self.available_options
                                        env_name: "FL_SONAR_RUNNER_ARGS",
                                        description: "Pass additional arguments to sonar-scanner. Be sure to provide the arguments with a leading `-D` e.g. FL_SONAR_RUNNER_ARGS=\"-Dsonar.verbose=true\"",
                                        optional: true),
-          FastlaneCore::ConfigItem.new(key: :sonar_login,
```

Comment:
We should probably consider keeping both options to not break user's configurations.

If we do, the old option should be deprecated. The 2 options should conflict each other, and the old option should set the value of the new one when it isn't set.

Reply:
Thanks @lacostej for the review! How would you want to mark options as deprecated, do you have a previous PR I could take a quick look at please?

Reply:
I think I addressed it, but will wait to resolve it until you confirm it of course :).